packages/contracts/contract.hbs

```diff
@@ -0,0 +1,76 @@
+## `{{name}}`
```

Comment:
don't they have a standard template ? that looks pretty generic.

Reply:
or is it copied from OZ documentation itself?

Reply:
It is copied, they don't present enums in the default template, I think.